mirror.ts

```diff
@@ -12,10 +13,22 @@ async function handleRequest(request) {
 	options.cf.image.height = 400
 
 	const imageURL = url.searchParams.get("img")
-
+	
+	const cache = caches.default
+	const cacheUrl = new URL(request.url)
+	cacheUrl.pathname = "/snapshot" + imageURL
```

Comment:
What's the need of "snapshot" here? 
I think it can be just
```
const cacheUrl = new URL(request.url)
const cacheKey = new Request(cacheUrl.toString(), request)
```
Like in here: https://developers.cloudflare.com/workers/examples/cache-api
The params with method: GET is used to force GET on POST requests, but this worker will only receive GET requests.